RUFAS/routines/manure/constants_and_units/manure_constants.py

```diff
@@ -71,3 +71,8 @@ class ManureConstants:
     """
     The fraction of total solids in Liquid manure. Temporary placeholder until upstream solids tracking is fixed.
     """
+
+    METHANE_DESTRUCTION_EFFICIENCY = 81
+    """
+    The fraction of methane destruction efficiency rate.
```

Comment:
```suggestion
    The percentage of methane destroyed on a daily(?) basis in systems using a cap and flare.
```
This constant is a percentage, not a fraction. Also, I am not sure that this is exactly what/how this constant used.

Reply:
I am not sure of the best description either. @KFosterReed do you have any suggestions?

Reply:
Ed's suggestion is great. I just removed 'daily' because although we are applying it daily, it is not exclusively a daily estimate